plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/SubversionClientServiceImpl.java

```diff
@@ -153,12 +155,14 @@ private String getBaseUrl() {
     }
 
     @Override
-    public Promise<InfoResponse> info(Path project, Path target, String revision, boolean children) {
+    public Promise<InfoResponse> info(Path project, Path target, String revision, boolean children, String userName, String password) {
```

Comment:
It would be better to avoid copying of code and reuse this method by info(Path, String, String, boolean)